src/richie/apps/courses/templates/courses/cms/fragment_organization_main_logo.html

```diff
@@ -1,17 +1,19 @@
 {% load cms_tags static %}{% spaceless %}
-{% if current_page.publisher_is_draft %}
-    {% if organization.check_publication is True %}
+{% if organization %}
```

Comment:
Have you thought of putting this "if" in the course_detail.html template to wrap the call to this fragment?

Reply:
Not really... is it better in your opinion ?

Reply:
i feel that because the fragment is called this way:
`{% include "courses/cms/fragment_organization_main_logo.html" with organization=course.get_main_organization %}`
that the check has to be done after. But then again we could call `if course.get_main_organization` before the include.

Reply:
Yeah you're right. By the way I think the main organization will never by null because it is a required field in the creation wizard. WDYT?

Reply:
It's just in case someone goes and delete the organization plugin. Especially if there's only one.